test/silhouette.jl

```diff
@@ -34,4 +34,18 @@ c = [3, 1]
 
 @test silhouettes(a, c, D) ≈ [0.5, 0.5, -1/3, 0.0]
 
+a = [fill(1, 5); fill(2, 5)]
+d = fill(1, (10, 10))
```

Comment:
I think the current version gives correct answer for `d[i, j] = 1`, the bug is about `d[i, j] = 0`.
```suggestion
d = fill(0, (10, 10))
```